osu.Game/Screens/Tournament/Drawings.cs

```diff
@@ -47,11 +47,18 @@ public class Drawings : OsuScreen
 
         public ITeamList TeamList;
 
+        protected override DependencyContainer CreateLocalDependencies(DependencyContainer parent) => new DependencyContainer(parent);
+
         [BackgroundDependencyLoader]
-        private void load(TextureStore textures, Storage storage)
+        private void load(TextureStore textures, Storage storage, DependencyContainer dependencies)
         {
             this.storage = storage;
 
+            TextureStore flagStore = new TextureStore(new RawTextureLoaderStore(new NamespacedResourceStore<byte[]>(new StorageBackedResourceStore(storage), "Drawings")));

```

Comment:
I'm not sure how you mean...

```
var storageStore = new StorageBackedResourceStore(storage);
TextureStore flagStore = new TextureStore(new ...(new...(storage, "Drawings")))
```
?
In this case I can't do e.g.
```TextureStore flagStore = new ...(new ...(new ... ...))
flagStore.AddStore(new StorageBackedResourceStore(storage));
``` because it's gonna be at the wrong level.